cmake/AddGHelpTarget.cmake

```diff
@@ -48,29 +39,47 @@ function (add_ghelp_target docname lang entities figures dtd_files)
     add_custom_command(
         OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-preparedir-trigger"
         COMMAND ${CMAKE_COMMAND} -E make_directory "${BUILD_DIR}/figures"
-        COMMAND ${CMAKE_COMMAND} -E make_directory "${BUILD_DIR}/images/callouts"
+        COMMAND ${CMAKE_COMMAND} -E make_directory "${BUILD_DIR}/images"
         COMMAND touch "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-preparedir-trigger")
 
-
+    # Copy DTD files for this document
     add_custom_command(
-        OUTPUT ${dest_files}
-        COMMAND ${CMAKE_COMMAND} -E copy ${source_files} "${BUILD_DIR}"
-        DEPENDS ${entities} "${docname}.xml" ${dtd_files}
-                "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-preparedir-trigger"
-        WORKING_DIRECTORY "${BUILD_DIR}")
+        OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-dtd-trigger"
+        COMMAND ${CMAKE_COMMAND} -E copy ${dtd_files} "${BUILD_DIR}"
+        COMMAND touch "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-dtd-trigger"
+        DEPENDS "${dtd_files}"
+                "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-preparedir-trigger")
```

Comment:
Don't use the detour of  xyz-trigger to simply copy files in place. Use  ` configure_file` instead.  More importantly though the extra separation of docbook files from dest_files doesn't fit in this commit. For starters I don't really see what you gain by doing so and if there really is gain it's a separate topic so it should be done in a separate commit.